### PR TITLE
created sahityaLMaruvada.md file

### PR DESCRIPTION
**Pull request by sahityaLMaruvada**
Included the needful details and brief description about me
[Rawgitlink](https://rawgit.com/sahityaLMaruvada/sahityaLMaruvada.github.io/sahityamaruvada-branch-1/#!pages/profiles/sahityaLMaruvada.md)

### CODE REVIEW

File: pages/profiles/sahityaLMaruvada.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+Python  | Drupal
+Java  |  Ruby On Rails
+HTML,CSS| Wordpress
+Ruby|
+
+I am very much interested in exploring the domain of Cloud Computing. My personal favourite hobbies include Traveling, Cooking.
+
+---
+Please feel free to connect with me on  
+*[LinkedIn](https://www.linkedin.com/in/lalitha-sahitya-maruvada/)
```
Comment (mappuji) [inline pages/profiles/sahityaLMaruvada.md]: You did a great job, however, in order to create a bullet point, you need to add space after the `*` symbol 😉

**Comment by duongdo27** [comment]
@prajwalajayaprakash How can you merge a pull request when I still request him to change?

**Comment by prajwalajayaprakash** [comment]
@duongdo27 I saw that the changes you had requested were done hence I merged it. Will not do it again.

**Comment by duongdo27** [comment]
I see.

**Comment by sahityaLMaruvada** [comment]
@duongdo27  can you please approve the change request. I have fixed the bullets issue in rawgit as per your suggestion.

**Comment by duongdo27** [comment]
@prajwalajayaprakash merge your PR already. It is fine for you now.

**Pull request merged**